.github/workflows/java-ci.yml

```diff
@@ -91,7 +91,11 @@ jobs:
           if [ -f "mvnw" ]; then
             ./mvnw deploy -Dpackaging=docker -Djib.from.image=ghcr.io/graalvm/graalvm-ce:java11-21.1.0 -Ddocker.image.suffix=graalvm -Dproject.version=$GITHUB_SHA --no-transfer-progress
           elif [ -f "gradlew" ]; then
-            ./gradlew dockerPush -Pversion=$GITHUB_SHA --no-daemon
+            if [ "${{ matrix.project-path }}" == "catalogue" ]; then
+              ./gradlew :app:dockerPush -Pversion=$GITHUB_SHA --no-daemon
+            else
+              ./gradlew dockerPush -Pversion=$GITHUB_SHA --no-daemon
+            fi
```

Comment:
If you do `gradle dockerPush`, then it will execute _all_ `dockerPush` tasks in _all_ projects. So if `:app` defines one, it will be called, but if `:aws` _also_ defines one, then it will be pushed too. There's no need to create a top level `dockerPush`. I think something clearer would be to move the top level `src` directory into its own subproject. That's how "idiomatic Gradle" recommends to setup projects, with the TLP being basically an empty shell, and subprojects defining components.